src/BeamAdapter/component/mapping/AdaptiveBeamMapping.cpp

```diff
@@ -195,6 +195,22 @@ void AdaptiveBeamMapping<Rigid3Types, Rigid3Types >::computeJacobianOnPoint(unsi
     dmsg_info()<<" ********** TEST J-Jt(transposed): ********** \n"<<Test;
 }
 
+
+template <>
+SOFA_BEAMADAPTER_API int AdaptiveBeamMapping<Rigid3Types, Rigid3Types >::addPoint (const Coord& point, int indexFrom)
```

Comment:
Why do you need the specialization?

If I look at the generic method, it uses `getCPos` which returns `getCenter`. So it looks to me that the generic methods support the rigid types.

Reply:
I dont know, I guess  I added it to try to solve some unreferenced things from MSVC....
I will remove it, as you said it is not necessary